### ISSUE TITLE
levensthein in the levensthein crate is faster

### ISSUE THREAD

**Issue by vks** [dguo/strsim-rs#15]
I benchmarked `levenshtein("kitten", "sitting")`, and those were the results:

```
test bench_levenshtein  ... bench:         201 ns/iter (+/- 10)
test bench_levenshtein2 ... bench:          92 ns/iter (+/- 3)
```

The first one uses `strsim::levenshtein`, the second one use [`levenshtein::levensthein`](https://github.com/wooorm/levenshtein-rs/blob/873844746aa069872e62f90d5c49f504b2ecc4bb/src/lib.rs#L8).

**Comment by dguo** [comment]
Yeah, I'm not surprised. There is definitely room for optimization that I haven't gotten around to. I'll try to give it some time in the next few weeks. Thanks for reporting it!

**Comment by dguo** [comment]
This should be resolved by #16.
